src/tree_builder/rules.rs

```diff
@@ -770,7 +770,7 @@ impl<Handle, Sink> TreeBuilderStep
                     Done
                 }
 
-                // FIXME: This should be unreachable, but match_token! requires a
+                // FIXME: This should be unreachable, but match_token requires a
```

Comment:
Note: this comment referred to my hand-written parser. `syn` doesn’t have this problem.